README.md

```diff
@@ -260,6 +260,10 @@ import Swipeable from 'react-native-gesture-handler/Swipeable';
  - `onSwipeableRightOpen` – method that is called when right action panel gets open.
  - `onSwipeableOpen` – method that is called when action panel gets open (either right or left).
  - `onSwipeableClose` – method that is called when action panel is closed.
+ - `onSwipeableLeftStartOpen` – method that is called when left action panel starts getting open.
+ - `onSwipeableRightStartOpen` – method that is called when right action panel starts getting open.
+ - `onSwipeableStartOpen` – method that is called when action panel starts getting open (either right or left).
```

Comment:
Can you rename "startOpen" to "willOpen" and also update the description here that it refers to the opening animation. Otherwise it is unclear if the callback will get called whenever the gesture starts, e.g. when the swipeable is closed and you start panning